include/lib/coreboot/coreboot_table.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2017, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __COREBOOT_TABLE_H
```

Comment:
I'll write them as David suggested for now, and then you can bulk edit all headers later if you need.